package-lock.json

```diff
@@ -2794,7 +2794,7 @@
     },
```

Comment:
Your package-lock.json changes just change `https` to `http`. You should remove those changes from your commit. (And check ` npm config get registry` is set correctly to `https://registry.npmjs.com/`)

Reply:
I have removed this change